Integrations/integration-Cybereason.yml

```diff
@@ -526,7 +532,8 @@ script:
                 'Suspects':suspects_string,
                 'LastUpdateTime': malop_last_update_time,
                 'Status': management_status,
-                'AffectedMachines': affected_machines
+                'AffectedMachine': affected_machines,
```

Comment:
why the machines was changed to machine? (single)

Reply:
Our context standard is singular